zebedee-cms/src/main/java/com/github/onsdigital/zebedee/service/ServiceStoreImpl.java

```diff
@@ -10,9 +10,14 @@
 import java.nio.file.FileAlreadyExistsException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.regex.Pattern;
+
+import static com.github.onsdigital.zebedee.logging.CMSLogEvent.warn;
 
 public class ServiceStoreImpl implements ServiceStore {
 
+    private static final Pattern VALID_FILENAME_REGEX = Pattern.compile("^\\w+$");
```

Comment:
Full disclosure - I suggested the `\w` used here, but that accepts underscores (as well as alphanumeric), which was not what Dave asked for - sorry.